bigquery_etl/dryrun.py

```diff
@@ -111,52 +107,100 @@
 }
 
 
-def dry_run_sql_file(sqlfile):
-    """Dry run the provided SQL file."""
-    sql = open(sqlfile).read()
+class DryRun:
+    """Dry run SQL files."""
+
+    DRY_RUN_URL = (
+        "https://us-central1-moz-fx-data-shared-prod.cloudfunctions.net/"
+        "bigquery-etl-dryrun"
+    )
+
+    def __init__(self, sqlfile):
+        """Instantiate DryRun class."""
+        self.sqlfile = sqlfile
+        self.dry_run_result = None
+
+    def _execute(self):
+        """Dry run the provided SQL file."""
+        if self.dry_run_result:
```

Comment:
I can't remember if the base python version here is 3.7 or 3.8. If 3.8, dry_run_result seems like a prime candidate to be expressed as a [cached property](https://docs.python.org/dev/library/functools.html#functools.cached_property), eliminating the need for `_execute`